code/game/gamemodes/miniantags/abduction/machinery/experiment.dm

```diff
@@ -181,8 +183,8 @@
 				to_chat(H, "<span class='warning'>You feel intensely watched.</span>")
 		sleep(5)
 		to_chat(H, "<span class='warning'><b>Your mind snaps!</b></span>")
-		var/objtype = pick(subtypesof(/datum/objective/abductee/))
-		var/datum/objective/abductee/O = new objtype()
+		var/objtype = (prob(72) ? /datum/objective/abductee/random : pick(subtypesof(/datum/objective/abductee/)))
+		var/datum/objective/abductee/O = new objtype() //45 possible objectives as of the time of this. 40 (88%) are under random. 90% - [1/6 * 100]% ~= 73%
```

Comment:
Keep in mind that the second half of the assignment of objtype can also pick random or any subtype of random as well.